docs/stdlib/math.rst

```diff
@@ -93,7 +92,7 @@ Math
 
     :index: logarithm
 
-    Return the base 10 logarithm of the input value.
+    Returns the Base 10 logarithm of ``x``:
```

Comment:
```suggestion
    Returns the base-10 logarithm of ``x``:
```

capitalizing Base isn't standard - I think hyphenation is most common: https://en.wikipedia.org/wiki/Decimal

Reply:
Base 10 is referenced without a hyphen as far as I can tell, but the capital letter may be unnecessary.

Reply:
It shouldn't be capitalized and it should probably be hyphenated. (https://docs.python.org/3/library/math.html#math.log10, for an example)

The general rule here (which is not universally followed) is that a multi-word phrase gets hyphenated when it is being used as an adjective. So you would write "base-10 logarithm" but "logarithm using base 10".

This does technically mean that you ought to hyphenate "peanut-butter-and-jelly sandwich" but few would take it that far.

Reply:
"Logarithm of base 10 from `x`" may be the simplest way to word this

Reply:
I'd stick with base-10 logarithm